.github/workflows/nodejs.yml

```diff
@@ -19,7 +19,8 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
       - run: npm install -g yarn
-      - run: yarn install --ignore-scripts --frozen-lockfile --ignore-engines
+      - run: yarn install --ignore-scripts --ignore-engines
+      - run: test -z "$(git diff)" || (echo 'Please run yarn and commit all changes to yarn.lock'; false)
```

Comment:
This will not work with windows. But shouldn't stop this from merging.

I'll make a note of it in the [windows/runner](https://github.com/trufflesuite/truffle/tree/windows/runner) branch

Reply:
Ooh, thanks, I wasn't thinking about that.  Good catch.  But yes we should definitely merge this anyway for now.